File: BriefAssistant/Models/BriefInfo.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace BriefAssistant.Models
{
    public class BriefInfo
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        [Required]
        public ContactInfo ContactInfo { get; set; }
        [Required]
        public CircuitCourtCase CircuitCourtCase { get; set; }
        public string AppellateCourtCaseNumber { get; set; }
        public string Argument { get; set; }
        public string Conclusion { get; set; }
        public BriefType Type { get; set; }
    }
}

File: BriefAssistant/ClientApp/src/app/models/BriefInfo.ts
import { ContactInfo } from "./ContactInfo";
import { CircuitCourtCase } from "./CircuitCourtCase";

export class BriefInfo {
	id: string;circuitCourtCase: CircuitCourtCase;
	appellateCourtCaseNumber: string;
	appendixDocuments: string;
	conclusion: string;
	caseFactsStatement: string;
	argument: string;
	oralArgumentStatement: string;
	publicationStatement: string;
	issuesPresented: string;
  contactInfo: ContactInfo;
  title: string;

	constructor() {
    this.circuitCourtCase = new CircuitCourtCase();
    this.appellateCourtCaseNumber = "";
		this.appendixDocuments = "";
		this.conclusion = "";
		this.caseFactsStatement = "";
		this.argument = "";
		this.oralArgumentStatement = "";
		this.publicationStatement = "";
		this.issuesPresented = "";
    this.contactInfo = new ContactInfo();
	  this.title = "";
	}
}
